docs/current/742989-cookbook.md

```diff
@@ -173,6 +173,85 @@ The following code listing builds a single image for different CPU architectures
 
 [Learn more](./guides/406009-multiplatform-support.md)
 
+### Build image from Dockerfile
+
+The following code listing builds an image using an existing Dockerfile.
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./quickstart/snippets/build-dockerfile/main.go
+```
+
+</TabItem>
+<TabItem value="Node.js">
+
+```javascript file=./quickstart/snippets/build-dockerfile/index.mjs
+```
+
+</TabItem>
+<TabItem value="Python">
+
+```python file=./quickstart/snippets/build-dockerfile/main.py
+```
+
+</TabItem>
+</Tabs>
+
+[Learn more](./quickstart/429462-quickstart-build-dockerfile.mdx)
+
+### Add OCI annotations to image
+
+The following code listing adds OpenContainer Initiative (OCI) annotations to an image.
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./cookbook/snippets/oci-annotations/main.go
+```
+
+</TabItem>
+<TabItem value="Node.js">
+
+```javascript file=./cookbook/snippets/oci-annotations/index.mjs
+```
+
+</TabItem>
+<TabItem value="Python">
+
+```python file=./cookbook/snippets/oci-annotations/main.py
+```
+
+</TabItem>
+</Tabs>
+
+### Define build-time variables
```

Comment:
These examples seem to have too much unrelated context. It might resonate more with the audience if we gave the `prod` example. Maybe this could be the first recipe that references [a Phoenix app](https://www.phoenixframework.org/)?

Step 1: https://hexdocs.pm/phoenix/up_and_running.html + [prereq.](https://hexdocs.pm/phoenix/installation.html)
Step 2:  https://hexdocs.pm/phoenix/deployment.html

Happy to help!

Reply:
I agree but I think this is also fine for the moment. I'm happy to change it in a follow-up PR if needed.